### ISSUE TITLE
Rolling over to next hour

### ISSUE THREAD

**Issue by Xytreyum** [burgerga/shinyTime#10]
Great addition to R/Shiny! 
Could it be possible to have the time rolling over to the next hour? For example, if it is `10:59` and you press up (to advance a minute) it would go to `11:00` (instead of doing nothing)? That would make it feel more natural. Thanks!

**Comment by burgerga** [comment]
Yes, I would love this as well, but I would need to completely rewrite the javascript :(
Adding it to my long-term todo list ;)

**Comment by Xytreyum** [comment]
I was afraid something like that would be the answer. Anyway, thanks for the consideration and I'll keep an eye on the project :)

**Comment by burgerga** [comment]
I'll keep it open to remind me because it is a nice feature to have.
